testapi.pm

```diff
@@ -2250,4 +2251,48 @@ sub backend_get_wait_still_screen_on_here_doc_input () {
     return get_var(_WAIT_STILL_SCREEN_ON_HERE_DOC_INPUT => $ret);
 }
 
+=head2 stash
+
+    stash key_name => $data;
+    my $data = stash 'key_name';
+
+Provides access to a Key/Value storage. Can be used to share data between
+different test modules, that is not suitable to save via C<set_var>.
+You can save any kind of data structure.
+
+=cut
+
+my %stash;
+sub stash ($key, @value) {
+    return $stash{$key} unless @value;
```

Comment:
Considering this only supports one value per key:
```suggestion
sub stash ($key, $value = undef) {
    return $stash{$key} unless defined $value;
```

Reply:
I was thinking that this should support storing undefined values.
And in that case using an array in the signature is the only way to check if a second parameter was given or not...